cypress/integration/components/check-category.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("Testa comportamento do componente Check Category", () => { 
+    it("Retorna se o componente está visível", () => {
+        cy.visit('/')
+        cy.get('.styles_container__MnfE_').should("be.visible")
+        cy.get('.styles_container__MnfE_ label').should("be.visible")
+        cy.get('.styles_categoryName__14xRg').should('be.visible')
+
+    })
+
+    it("Retorna se o componente tem uma classe chamada isChecked", () => {
+        cy.get('.styles_container__MnfE_ label input').should('have.class', 'styles_isChecked__m-a4l')
+    })
+
+    it("Retorna se, após clicar no botão, a classe isChecked some", () => {
+        cy.get('.styles_container__MnfE_ label').click()
+        cy.get('.styles_isChecked__m-a4l').should('not.exist')
```

Comment:
essa função está verificando que a classe especificada não existe na tela inteira, então se futuramente você tiver um elemento com essa classe, o teste pode falhar, uma outra abordagem poderia ser:

```suggestion
cy.get('.styles_container__MnfE_.label').should('not.have.class', 'styles_isChecked__m-a4l')
```